packages/components/src/AnimatedSwitcher/transitions.ts

```diff
@@ -22,24 +22,24 @@ export const SPIN_COUNTER_CLOCK_WISE: Variants = {
   visible: {
     rotate: 0,
     scale: 1,
-    transition: { duration: DURATION_AVERAGE, ease: "easeOut" },
+    transition: { duration: DURATION_SIMPLE, ease: "easeOut" },
```

Comment:
Cutting it down to 0.1 in each direction gets us to a total of 0.2 which matches our `timing-base` token.